### PR TITLE
Expand Macro Content is Not Displayed in a PDF Export #246

### PR DESCRIPTION
**Pull request by Farcasut**
* Added a check to see if the page is in HTML print view; if we are in that view, we open the macro.

### CODE REVIEW

**Comment by Farcasut** [comment]
Preview: 
image

File: xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml [hunk 1]
```diff
@@ -410,9 +410,9 @@ Hello 👀
 #macro (executeMacro)
   $xwiki.ssx.use('Confluence.Macros.Expand')
   #set ($opened = $xcontext.action == 'edit')
```
Comment (trrenty) [inline xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml:412]:
```suggestion
  #set ($opened = $xcontext.action == 'edit' || $request.getParameter('format') == "html-print")
```

File: xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml [hunk 2]
```diff
@@ -410,9 +410,9 @@ Hello 👀
...
-
+  #set ($export = $request.getParameter('format'))
   {{html clean="false" wiki="true"}}
-  &lt;details class="confluence-expand-macro panel panel-default" #if ($opened)open#end&gt;
+  &lt;details class="confluence-expand-macro panel panel-default" #if ($opened || $!export == "html-print")open#end&gt;
```
Comment (trrenty) [inline xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml:415]:
```suggestion
  &lt;details class="confluence-expand-macro panel panel-default" #if ($opened)open#end&gt;
```

**Review by trrenty — COMMENTED** [review]
Also. Did you check both the "ctrl + p" function of the browser (that prints the page) and the pdf export functionality of XWiki? I dont think the first one is important for us. We are more interested in the pdf export.

File: xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml (hunk 2 above)
Comment (Farcasut) [reply xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml]: The Ctrl-P doesn't work, but the PDF export from XWiki works as intended now

**Comment by Farcasut** [comment]
The Ctrl-P doesn't work, but the PDF export from XWiki works as intended now

**Pull request merged**